sdelete.txt

```diff
@@ -0,0 +1,20 @@
+[Section]
+Name = SDelete
+Version = 2.02
+License = Sysinternals Software License Terms
+Description = SDelete is a free command line tool that you can use to delete files securely so that they cannot be recovered anymore. You will need a ZIP decompression program to install it.
+Size = 222 KiB

```

Comment:
You can get rid of this "Size" field, which is deprecated; the SizeInBytes field from below is better.

Reply:
I kept that because wiki has this warning about Size section. 

"DEPRECATED: The value should be kept for backward compatibility."
As you say I can get rid of it, then I'll do it asap.

Reply:
It would be backward compatibility with older versions of RAPPS that you get only by downloading old versions of ReactOS.

Reply:
Thanks, I fixed the wiki article

Reply:
Anything else?